### ISSUE TITLE
cannot uninstall

### ISSUE THREAD

**Issue by hyphone** [andrejilderda/joplin-macos-native-theme#50]
Hello and good day!

I'm not able to remove the plugin from Joplin.
When I try to remove it from the Extensions-Tab, I'm asked to restart Joplin, what I also do.

After restarting Joplin, the plugin is still there but as an older version and I can update or remove it again.

When I want to remove it it is again still there.

I also tried to delete the folders in ~/.config/joplin-desktop/cache and ../plugins without success.

All the best

**Comment by andrejilderda** [comment]
@hyphone Hmm... that's weird! You don't have the theme added as a 'Development plugin', do you?
![screenshot 2022-01-17 at 11 52](https://user-images.githubusercontent.com/487182/149756632-2136c4a6-20a8-439e-ba07-7f34f4e86281.png)

**Comment by hyphone** [comment]
Thank you for having a look at it so quickly!

I'm currently not sure. It could be that I did so in the very beginning.

**Comment by hyphone** [comment]
Can't find more in the console than:

`PluginService: Could not find plugin path to uninstall - nothing will be done: com.andrejilderda.macOSTheme`

**Comment by andrejilderda** [comment]
I've double checked the id of the theme and it hasn't changed, so that can't be it. Can you maybe share what the `id` is in the `manifest.json` in Joplin's cache folder? On macOS this is:
`/Users/<YOUR USERNAME>/.config/joplin-desktop/cache/com.andrejilderda.macOSTheme/manifest.json`

**Comment by hyphone** [comment]
I don't have any files like this anymore in my home folder.
When I close Joplin the only file left is `~/.config/joplin-desktop/tmp/plugin_com.andrejilderda.macOSTheme.js`
But when I also delete that `v1.2.10` is still installed somehow and the `~/.config/joplin-desktop/tmp/plugin_com.andrejilderda.macOSTheme.js` file is there again (but no other files, none in cache and none in plugins).
I'm a bit clueless :sweat_smile:

**Comment by hyphone** [comment]
I like the theme very much - I just wanted to try something with the default theme and a custom CSS and recognised this.

**Comment by andrejilderda** [comment]
That indeed is super weird. It seems like a bug in Joplin. I think it's worth investigating further, for that the Joplin repo is the best place: https://github.com/laurent22/joplin/issues 

Glad to hear you enjoy the theme! 🙃

**Comment by hyphone** [comment]
I think it had something to do with the manual installation (installed as a development plugin as you suggested).
At first I installed it when I think there was no possibility to install plugins within Joplin (or I did not know at the time because I wasn't aware of the plugins for quite some time). So I manually added it. And perhaps that's the version that it always jumped back to when I wanted to uninstall it.

I reset my Joplin settings, installed the theme with Joplin, uninstalled it and it is successfully uninstalled.

As a happy user, I installed it again of course :-D

So I think you might close this issue. I sadly have no intention to post it in the Joplin notes as I think Laurent has so much to do - I don't want to post a more or less trivial thing the probably won't happen to others in the future.
If I encounter such an issue again I for sure will notify Joplin about it.

Thank you though for having a look and the super quick responses.
